src/MultiTypeSymEntry.chpl

```diff
@@ -439,6 +451,13 @@ module MultiTypeSymEntry
     proc toGenSymEntry(entry: borrowed AbstractSymEntry) throws {
         return (entry: borrowed GenSymEntry);
     }
+
+    /**
+     * Helper proc to cast AbstrcatSymEntry to CompositeSymEntry
+     */
+    proc toCompositeSymEntry(entry: borrowed AbstractSymEntry) throws {
+        return (entry: borrowed CompositeSymEntry);
+    }
     
     /**
      * Helper proc to cast AbstrcatSymEntry to SegStringSymEntry
```

Comment:
I know this isn't part of the changes, but since it's the same typo as above I figured I would mention it here as well

Reply:
fixed